gradle/libs.versions.toml

```diff
@@ -3,7 +3,7 @@ micronaut = "4.0.0-M2"
 micronaut-platform = "4.0.0-SNAPSHOT"
 micronaut-docs = "2.0.0"
 micronaut-gradle-plugin = "4.0.0-M2"
-micronaut-testresources = "2.0.0-M3"
+micronaut-testresources = "2.0.0-M4"
```

Comment:
Do we need to update testresources version in the gradle plugin too?

Reply:
We don't have to because we override the version in https://github.com/micronaut-projects/micronaut-data/blob/master/data-jdbc/build.gradle#L86

I will release a new Gradle plugin version once a new version of the platform is released.